demo-simple/src/main/java/com/novoda/downloadmanager/demo/CustomCallbackThrottle.java

```diff
@@ -2,16 +2,16 @@
 
 import com.novoda.notils.logger.simple.Log;
 import com.novoda.downloadmanager.CallbackThrottle;
-import com.novoda.downloadmanager.DownloadBatchCallback;
+import com.novoda.downloadmanager.DownloadBatchStatusCallback;
 import com.novoda.downloadmanager.DownloadBatchStatus;
 
 // Must be public
 public class CustomCallbackThrottle implements CallbackThrottle {
 
-    private DownloadBatchCallback callback;
+    private DownloadBatchStatusCallback callback;
```

Comment:
Sorry, the fact that it handled `DownloadBatchStatus` but was called a `DownloadBatchStatusCallback` was irritating me 😄

Reply:
* was called `DownloadBatchCallback ` (so you have renamed it to `DownloadBatchStatusCallback `)